.gitignore

```diff
@@ -26,3 +26,4 @@ z_build
 .DS_Store
 outputDir/
 **/Version.java
+**/bin/
```

Comment:
Why do we need that change?

Reply:
When building TestNG on my local machine, thousands of files were generated in the **/bin/* folders that should not be committed back in, mostly Kotlin files.